services/api/src/models/definitions/user.json

```diff
@@ -60,11 +60,18 @@
       "type": "Date",
       "writeScopes": "none"
     },
-    "authTokenId": {
-      "type": "String",
-      "readScopes": "none",
-      "writeScopes": "none"
-    },
+    "authTokens": [
```

Comment:
@andrewplummer how can I put the 
``` 
"readScopes": "none",
"writeScopes": "none"
```

on authTokens, on the full array not just the keys

Reply:
You should be able to do:

```
"authTokens": {
   "type": "String",
   "readScopes": "none",
   "writeScopes": "none",
}
```